### ISSUE TITLE
restPrefix attribute doesn't work

### ISSUE THREAD

**Issue by Beingbook** [balderdashy/sails#3091]
REST API is foolish when 'restPrefix' attribute edited such as '/api'. Original and namespace both don't works. e.g) /users, /api/users are away from server.

**Comment by tjwebb** [comment]
> Original and namespace both don't works. e.g) /users, /api/users are away from server.

That's correct. If you use restPrefix, then your requests now require that prefix.

**Comment by Beingbook** [comment]
@tjwebb What I meant was that accessing with prefix or without prefix API is disable.

**Comment by sailsbot** [comment]
Thanks for posting, @Beingbook.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
